baselines/fedrep/fedrep/server_app.py

```diff
@@ -0,0 +1,77 @@
+"""Server implementation."""
+
+import json
+import os
+import time
+from typing import List, Tuple
+
+from fedrep.utils import get_create_model_fn, get_server_strategy
+from flwr.common import Context, Metrics, ndarrays_to_parameters
+from flwr.server import ServerApp, ServerAppComponents, ServerConfig
+
+# Initialize the execution results directory.
+RES_SAVE_PATH = "./results"
+if not os.path.exists(RES_SAVE_PATH):
+    os.makedirs(RES_SAVE_PATH)
+RES_SAVE_NAME = time.strftime("%Y-%m-%d-%H-%M-%S")
+# Set the file name where the results will be saved.
+RES_FILE = f"{RES_SAVE_PATH}/{RES_SAVE_NAME}.json"
```

Comment:
Could we wrap this into a function. Generally we try not to define things globally because, although it's fine in this case, if that become the norm in other parts of the code (e.g. in the `ClientApp`) because of how the ephemeral processes work, things might not behave as one might think at first.

Reply:
Kept `RES_FILE` as the only global variable and moved the directory and file creation inside a function. A new evaluation result is written inside the `RES_FILE` through the `write_res()` function.